weaviate/collection/classes/data.py

```diff
@@ -35,40 +36,23 @@ class _BatchReturn:
 
 @dataclass
 class RefError:
-    message: str
-
-
-@dataclass
-class ReferenceTo:
```

Comment:
are we not using this anymore?

Reply:
Yes, it's covered by `ReferenceFactory` now